### PR TITLE
opendatahub/vllm: include adapter smoke test for main

### PR DESCRIPTION
**Pull request by dtrifiro**
Enables grpc server smoke test for `main` branch

### CODE REVIEW

**Comment by dtrifiro** [comment]
Merge after https://github.com/openshift/release/pull/52543 and https://github.com/opendatahub-io/vllm/pull/3

File: ci-operator/config/opendatahub-io/vllm/opendatahub-io-vllm-main.yaml [hunk 1]
```diff
@@ -76,7 +78,35 @@ tests:
       localhost:8000/v1/completions | python -m json.tool || \
         (kill -9 $server_pid && exit 1)
 
-    echo "success"
+    echo "OpenAI API success"
```
Comment (terrytangyuan) [inline ci-operator/config/opendatahub-io/vllm/opendatahub-io-vllm-main.yaml:81]: Consider moving this to a script?

**Pull request merged**